test/integration.test.js

```diff
@@ -55,4 +68,39 @@ describe('Integration testing generateFromFile() to make sure the result of the
     const file = await readFile(path.join(outputDir, testOutputFile), 'utf8');
     expect(file).toMatchSnapshot();
   });
+
+  it('should ignore specified files with noOverwriteGlobs', async () => {
+    const logSpyDebug = jest.spyOn(log, 'debug').mockImplementation(() => {});
+    // log.debug = jest.fn(); 
+
+    const outputDir = generateFolderName();
+    // Manually create a file to test if it's not overwritten
+    if (!await exists(outputDir)) {
```

Comment:
actually I don't get why we need to validate if it exists, if a line earlier it is created

Reply:
```
const generateFolderName = () => {
    // you always want to generate to new directory to make sure test runs in clear environment
    // normalize the path for cross-platform compatibility
    // return path.normalize(path.resolve(mainTestResultPath, crypto.randomBytes(4).toString('hex')));
    return path.resolve(mainTestResultPath, crypto.randomBytes(4).toString('hex'));
  };
```

If I understand it correctly, `generateFolderName()` does not actually generate the directory, but just creates and returns a path string. Thus I manually checks and creates it.

Reply:
I see that in other tests you have written, you did not check and create the directory. Did you not have directory or path not found issue? If I comment out line 78-80, I have 
```
ENOENT: no such file or directory, open '/Users/yuanyuan/workspace/generator/test/temp/integrationTestResult/6224bfdd/test-file.md'
```